package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stytch",
-  "version": "10.11.0",
+  "version": "10.13.0",
```

Comment:
Wait lol what? Should be 10.12. Must've accidentally done `npm version minor` twice, you can just manually edit package.json and package-lock.json if needed (I think @jennifer-stytch dealt with this a couple weeks ago)